src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -253,7 +258,7 @@ public function chgrp($files, $group, $recursive = false)
     public function rename($origin, $target, $overwrite = false)
     {
         // we check that target does not exist
-        if (!$overwrite && is_readable($target)) {
+        if (!$overwrite && $this->isReadable($target)) {
```

Comment:
Well your question would be best addressed to fabpot, since he put this in at [17 Feb 2010](https://github.com/symfony/symfony/blob/5bf8908f6b4897c5f7424d43b2e3ce4418e03521/src/Symfony/Framework/WebBundle/Util/Filesystem.php#L173). But my guess is that `file_exists` would work just as well here